service/src/main/java/bio/terra/landingzone/library/landingzones/management/ResourcesReaderImpl.java

```diff
@@ -59,13 +53,43 @@ public List<DeployedResource> listResourcesByPurpose(ResourcePurpose purpose) {
         purpose.toString());
   }
 
+  @Override
+  public List<DeployedResource> listResourcesWithPurpose() {
+    List<ResourcePurpose> supportedPurposes = ResourcePurpose.values().stream().toList();
+    return listResourceByTag(
+            resourceGroup.name(), LandingZoneTagKeys.LANDING_ZONE_PURPOSE.toString(), null)
+        .stream()
```

Comment:
Minor - maybe the private `listResourcesByTag` could return a `Stream<DeployedResource>` so you don't need to convert to List, back to Stream, back to List?

Reply:
Good point, but I won't change it now.